content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -278,7 +296,11 @@ Alternatively, you can try separating the `key=value` pairs like so:
 but this will result in the key `enable-admission-plugins` only having the value of `NamespaceExists`.
 
 A known workaround is to use the kubeadm
-[configuration file](/docs/setup/independent/control-plane-flags/#apiserver-flags).
+<<<<<<< HEAD:content/en/docs/setup/production-environment/tools/independent/troubleshooting-kubeadm.md
```

Comment:
This looks like a merge left in by mistake.

Reply:
Thank you it was a miss. Fixed it now.